### DISCUSSION TITLE
trying to get started, languageCode errors

### DISCUSSION THREAD

**Discussion by ahill00** [jpanther/congo#712] [Q&A]
I am trying to get started with hugo/congo from the tutorials, and I keep running into this error:

```
panther/congo/v2@v2.7.4/layouts/_default/baseof.html:5:12": execute of template failed: template: _default/taxonomy.html:5:12: executing "_default/taxonomy.html" at <site>: can't evaluate field LanguageCode in type *langs.Language
```

my config directory looks like this:

```
andy@debian:~/code/ahill.net$ tree config/
config/
└── _default
    ├── config.toml
    ├── languages.en.toml
    ├── markup.toml
    ├── menus.en.toml
    ├── module.toml
    └── params.toml
```

and in config.toml I have 

```
defaultContentLanguage = "en"
languageCode = "en"
```

setup is:

```
hugo v0.111.3+extended linux/amd64 BuildDate=2023-03-16T08:41:31Z VendorInfo=debian:0.111.3-1
```

and I've installed congo from hugo mods. Any ideas?

**Comment by wolfspyre** [comment]
Please update your question :) According to your post, one is lead to believe you have no errors ;)

btw, latest hugo is 0.120.4, may want to try that for shiggles

**Comment by ahill00** [comment]
thanks, updated!

**Comment by ahill00** [comment]
upgrading worked. I had to manually uninstall from apt, and just install the .deb for 0.120.4. Thanks!
